hips/draw/tests/test_simple.py

```diff
@@ -1,102 +1,74 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 import pytest
 import numpy as np
-import healpy as hp
-from astropy.coordinates import SkyCoord
 from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
 from astropy.tests.helper import remote_data
-from ..simple import make_sky_image, draw_sky_image, compute_matching_hips_order, _get_hips_order_for_resolution
-from ...tiles import HipsSurveyProperties, HipsTileMeta, HipsTile
-from ...utils import WCSGeometry
-from ...utils.testing import get_hips_extra_file, make_test_wcs_geometry, requires_hips_extra
-
-
-def get_test_tiles(file_format, survey, order, ipix_list):
-    filename = get_hips_extra_file('datasets/samples/' + survey + '/properties')
-    hips_survey = HipsSurveyProperties.read(filename)
-
-    tiles = []
-    for ipix in ipix_list:
-        tiles.append(HipsTile.read(
-            meta=HipsTileMeta(order=order, ipix=ipix, file_format=file_format, frame=hips_survey.astropy_frame),
-            full_path=get_hips_extra_file('datasets/samples/' + survey + hips_survey.tile_path(order=order, ipix=ipix, tile_format=file_format)),
-        ))
-
-    return tiles
-
-
-draw_sky_image_pars = [
-    dict(file_format='fits', shape=(1000, 2000), survey='DSS2Red', data_1=2866.0101409848185,
-         data_2=2563.6916727348043, data_sum=4575235421.512643, order=3, ipix_list=[450, 451]),
-    dict(file_format='jpg', shape=(1000, 2000, 3), survey='DSS2Red', data_1=[13.040878, 13.040878, 13.040878],
-         data_2=[17.235874, 17.235874, 17.235874], data_sum=243177268.56158745, order=3, ipix_list=[450, 451]),
-    dict(file_format='png', shape=(1000, 2000, 4), survey='AKARI-FIS', data_1=[254., 254., 254., 255.],
```

Comment:
Why are you removing the PNG test here?

I looked at this earlier today, and I don't see why you couldn't just keep it!?
https://github.com/hipspy/hips-extra/issues/5#issuecomment-315030888